hadoop-common-project/hadoop-common/src/site/markdown/FileSystemShell.md

```diff
@@ -824,6 +824,19 @@ Example:
 * `hadoop fs -truncate 55 /user/hadoop/file1 /user/hadoop/file2`
 * `hadoop fs -truncate -w 127 hdfs://nn1.example.com/user/hadoop/file1`
 
+concat
+--------
+
+Usage: `hadoop fs -concat <target file> <source files>`
+
+Concatenate existing source files into the target file. Target file and source
+files should be in the same directory.
+
+Example:
+
+* `hadoop fs -concat /user/hadoop/target-file /user/hadoop/file-0 /user/hadoop/file-1`
+* `hadoop fs -concat /user/hadoop/target-file /user/hadoop/file-*`
```

Comment:
one of the examples should include the hdfs:// reference; for the -* one, maybe add  \* for those shells which do the expansion locally (i.e. bash), otherwise things don't work right

Reply:
I didn't understand with the expansion locally part. I think the expansion would be done by org.apache.hadoop.fs.shell.Command so even not using 'bash' the expansion would still work correctly? Sorry if I understand it wrong. A more detailed explanation would be very helpful.

Reply:
I mean the example needs to make clear if you don't escape the * with a \ then the shell will expand first

Reply:
Oh I see, thanks very much for your explanation ! In the example I want to show the user that he can use both the two commands. The results are the same. Both will concat target-file+file-0+file-1. Seems it makes the user even more confused. So let me just remove the second command.